src/v0/destinations/iterable/transform.js

```diff
@@ -120,7 +121,7 @@ const responseBuilderForRegisterDeviceOrBrowserTokenEvents = (message, destinati
  * @param {*} message
  * @returns
  */
-const getCategory = (messageType, message) => {
+const getCategory = (messageType, message, dataCenter) => {
   const eventType = messageType.toLowerCase();
```

Comment:
Let's remove `messageType` argument and do `message.type.toLowerCase()` instead.

Reply:
@sanpj2292 I checked the process function we are already passing messageType as :
`const messageType = message.type.toLowerCase()` the why we need to do this :
` const eventType = messageType.toLowerCase();`